app/services/sso/apache.rb

```diff
@@ -3,6 +3,7 @@ class Apache < Base
     delegate :session, :to => :controller
 
     CAS_USERNAME = 'REMOTE_USER'
+    OIDC_ACCESS_TOKEN = 'HTTP_OIDC_ACCESS_TOKEN'
```

Comment:
this could get out of sync with https://github.com/theforeman/foreman/blob/ef511e1c58c528e073a55c021a0c257d0b73b012/app/services/sso/openid_connect.rb#L50

you can extract this constant to https://github.com/theforeman/foreman/blob/ef511e1c58c528e073a55c021a0c257d0b73b012/app/services/sso/base.rb#L2 and reuse on both places I guess